client/cli/src/runtime.rs

```diff
@@ -86,7 +86,7 @@ where
 	T: AbstractService + Unpin,
 {
 	let informant_future = informant::build(&service);
-	let handle = async { informant_future.await };
+	async { informant_future.await };
```

Comment:
That should be just:
```suggestion
	informant_future.await;
```

Reply:
Maybe the easiest would be to simply use a `join!`

Reply:
I've removed `run_service_with_informant` and just put the code in `run_service_until_exit`